Add Avalonia.Markup.Declarative to MVVM & MVP & MVU

GavinHome opened the pull request.

OmidID commented:
@GavinHome Would you please rename it to Markup Declarative, then you need to move it because the list is sorted.

GavinHome commented:
> @GavinHome Would you please rename it to Markup Declarative, then you need to move it because the list is sorted.

I renamed to Markup Declarative and moved it.
